new/LIT6958Egziamlakfatari.xml

```diff
@@ -32,8 +32,8 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
         <profileDesc>
             <creation/>
             <abstract>
-                <p>This poem is referred to as qǝne of the type <foreign xml:lang="gez">kǝbr yǝʾǝti</foreign>,
-                    in <ref type="mss" corresp="ESdd037"/>, and attributed to <persName ref="PRS14409Yohannes"/>.</p>
+                <p>This poem is referred to as qǝne, in <ref type="mss" corresp="BNFabb57"/>, attributed to <persName ref="PRS14051LaekaMaryam"/>, 
+                    possibly of the type <foreign xml:lang="gez">ʿǝṭāna mogar</foreign>.</p>
```

Comment:
use now `<term key="EtanaMogar">ʿǝṭāna mogar</term>`

Reply:
here and elsewhere

Reply:
ok

Reply:
not yet?